code/api/src/config/database.json

```diff
@@ -1,12 +1,22 @@
 {
   "development": {
-    "username": "postgres",
+    "username": "skb",
```

Comment:
I realize we are going to have problems with this when we merge each others' branches but not sure how to handle it. Any ideas?

Reply:
Could we add it to our .gitignore file?

Reply:
Ooo interesting, let's give that a try. I'll go ahead and ignore this before we merge.